src/components/Navigation.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation: React.FC = () => {
+  return (
```

Comment:
when you dont need the function body you should just return straight away

Reply:
Done!